apps/storefront/src/components/addSku.tsx

```diff
@@ -0,0 +1,51 @@
+import { z } from 'zod';
+
+import B3Request from '@/shared/service/request/b3Fetch';
+import { LineItem } from '@/utils/b3Product/b3Product';
+
+const Options = z.array(
+  z.object({
+    optionEntityId: z.number(),
+    valueEntityId: z.number(),
+  }),
+);
+
+interface ProductsWithOptionSelections {
+  data: {
+    site: {
+      productWithSelectedOptions: {
+        sku: string;
+      };
+    };
+  };
+}
+
+export const addSku = async (product: LineItem): Promise<LineItem> => {
```

Comment:
the `add` makes it read a bit like a mutation 

`withSku`? Or `getSku` and let the client code do the tiny bit of merging?

Reply:
`getSku` it is